br/pkg/lightning/mydump/csv_parser.go

```diff
@@ -607,6 +607,9 @@ func (parser *CSVParser) ReadColumns() error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+	if parser.cfg.SkipHeader {
```

Comment:
can we move it earlier in case the first line contains bad CSV syntax? and use ReadUntilTernimator instead to skip one line

Reply:
I think we still need to parse the whole record, rather than stop at the terminator.  Suppose the header record covers several lines, if we just stop at the first line and count it as the header to skip.  Then the parsing will face syntax error problem.  But actually the CSV file is valid.